nautobot/docs/configuration/optional-settings.md

```diff
@@ -956,3 +933,36 @@ Environment Variables:
 * `NAUTOBOT_SHORT_TIME_FORMAT`
 * `NAUTOBOT_DATETIME_FORMAT`
 * `NAUTOBOT_SHORT_DATETIME_FORMAT`
+
+## Settings as Only Environment Variables
+
+The following settings are **only** configurable as Environment Variables, and not via `nautobot_config.py` or similar.
+
+---
+
+### GIT_SSL_NO_VERIFY
+
+Default: Unset
+
+Environment Variable: `GIT_SSL_NO_VERIFY`
+
+If you are using a self-signed git repository, you will need to set the environment variable `GIT_SSL_NO_VERIFY="1"`
+in order for the repository to sync.
+
+!!! warning
+    This _must_ be specified as an environment variable. Setting it in `nautobot_config.py` will not have the desired effect.
```

Comment:
I think this is also not needed after the reorganization.
```suggestion
```

Reply:
I would prefer to keep these 3 open ones in. While it is clear from the headings that each section is different, as the reader it is easy to lose that context when reading that specfic section and the lions share is training you to think differently. Not going to lose sleep, so feel free to commit suggested or resolve.